src/main/java/pb360/model/embedded/AnnotationParam.java

```diff
@@ -0,0 +1,30 @@
+package pb360.model.embedded;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AnnotationParam {
+	@JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
+	private String annotationParam;
```

Comment:
Why we need create a Class for containning one String variable?

Reply:
i thought i should use @jsonproperty for each line. I deleted this